<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoWidget</class>
 <widget class="QWidget" name="InfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>80</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="levelLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>60</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#00007f;&quot;&gt;关卡&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="levelLcdNumber">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>60</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
   <property name="numDigits">
    <number>3</number>
   </property>
   <property name="digitCount">
    <number>3</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Filled</enum>
   </property>
  </widget>
  <widget class="QLabel" name="stepLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>60</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#00007f;&quot;&gt;步数&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="stepLcdNumber">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>60</width>
     <height>35</height>
    </rect>
   </property>
   <property name="numDigits">
    <number>3</number>
   </property>
  </widget>
  <widget class="QPushButton" name="restartButton">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>270</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>重新开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="undoButton">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>320</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>退一步</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
